docs/security/imagemitigation.json

```diff
@@ -0,0 +1,232 @@
+{
+  "images": [
+    {
+      "registry": "gcr.io/k8s-prow/*",
+      "purpose" : "Prow CI",
+      "publisher": "Prow community",
+      "sourceCode": "https://github.com/kubernetes/test-infra/tree/master/prow",
+      "contributors": 1100,
+      "activity": {
+        "frequency": "high",
+        "evidence": "https://github.com/kubernetes/test-infra/commits/master/prow"
+      },
+      "comment": "",
+      "openSSF": {
+        "score": 5.6,
+        "review": {
+          "approvedImage": false,
+          "reason": ""
+        }
+      }
+    },
+    {
+      "registry": "docker.io/prom/*",
```

Comment:
The registry as well as the source code link points to a provider and not to specific images / repos. To analyze the risk introduced by a dedicated FOSS component, the absolute link to the image / repo is needed.

Reply:
Let me fix that ;)

Reply:
QQ... is this also required for the prow images?